crates/turborepo-lib/src/process/child.rs

```diff
@@ -362,26 +354,23 @@ impl Child {
         self.pid
     }
 
-    pub fn stdin(&mut self) -> Option<tokio::process::ChildStdin> {
+    fn stdin(&mut self) -> Option<tokio::process::ChildStdin> {
         self.stdin.lock().unwrap().take()
     }
 
-    pub fn stdout(&mut self) -> Option<tokio::process::ChildStdout> {
+    fn stdout(&mut self) -> Option<tokio::process::ChildStdout> {
         self.stdout.lock().unwrap().take()
     }
 
-    pub fn stderr(&mut self) -> Option<tokio::process::ChildStderr> {
+    fn stderr(&mut self) -> Option<tokio::process::ChildStderr> {
         self.stderr.lock().unwrap().take()
     }
 
     /// Wait for the `Child` to exit and pipe any stdout and stderr to the
-    /// provided writers.
-    /// If `None` is passed for stderr then all output produced will be piped
-    /// to stdout
+    /// provided writer.
     pub async fn wait_with_piped_outputs<W: Write>(
         &mut self,
         mut stdout_pipe: W,
-        mut stderr_pipe: Option<W>,
```

Comment:
Independently capturing stderr wasn't an used outside of tests and can't be easily done with processes that are hooked up to a pseudo-terminal (possible, but involves redirecting stderr to a file and streaming that). In preparation we remove this feature.